examples/index.md

```diff
@@ -126,3 +126,119 @@ Content-Type: application/vnd.api+json
 for readability. In practice, these characters must be percent-encoded, as
 noted in the base specification.
 
+## Error Objects
+
+Examples of how [example objects](http://jsonapi.org/format/#error-objects) work.
+
+### Responses to failed resource creation:
+
+Response with one error object:
+
+```http
+HTTP/1.1 422 Unprocessable Entity
+Content-Type: application/vnd.api+json
+
+{
+  "errors": [
+    {
+      "source": { "pointer": "data/attributes/first_name" },
```

Comment:
Although underscores are compliant, I'd prefer to use dasherized names, such as `first-name`, in examples as per our recommendations.